homepage/theme/templates/index.html

```diff
@@ -23,131 +23,501 @@ <h1 class="f1-ns f2 normal lh-solid ma0 mb3">
+
+            <div class="flex flex-wrap justify-center">
+              <div class="w-20 mt4">
+                <img src="/theme/images/gfx-compare-a.png" alt="numbers" width="75px" height="315px">
+              </div>
+              <div class="w-80 mt4">
+                <p class="ma0 ml3">
+                  Self host <i>Offen and thereby comply with GDPR guidelines.</i>
+                </p>
+                <p class="ma0 ml3 mt4">
+                  <i>Integrate the</i> code snippet <i>into pages you want to track.</i>
+                </p>
+                <p class="ma0 ml3 mt4">
+                  <i>Make your</i> users aware <i>of the access to their data.</i>
+                </p>
+                <p class="ma0 ml3 mt4 mb5">
+                  <i>Improve your services with</i> fair and transparent insights.
+                </p>
+              </div>
+            </div>
+
+          </div>
         </div>
+
       </div>
-    </div>
-  </div>
-{% endblock %}
 
-{% block outro %}
-<div class="w-100" id="bg-feature">
-  <div class="pa4">
-    <div class="mw8 center flex flex-column flex-row-ns mb5 pa3 cclr-mid-mint">
-      <div class="w-100 w-60-ns">
-        <h2 class="f1-ns f2 normal lh-title tc tl-ns ma0 mb3">
-          Test Offen today
-        </h2>
-        <p class="ma0 mb2 tc tl-ns">
-          Our alpha release is ready for testing. Offen is free and always will be.
-        </p>
-        <p class="ma0 mb3 tc tl-ns">
-          Be aware that things can still contain issues and that the upgrade path may be broken with upcoming releases.
-        </p>
+      <!-- COMPARE B -->
+      <div class="w-50-l w-100">
+
+        <div class="w-100 dn db-l h3 bg-white">
+        </div>
+
+        <div class="w-100 dib cclr-bg-mint-mid">
+          <div class="wx24 center">
+            <p class="f25 lh-solid ma0 mt5 mb3">
+              Benefits for your users
+            </p>
+          </div>
+        </div>
+
+        <div class="w-100 hx24 dib cclr-bg-white-fade">
+          <div class="wx24 center">
+
+            <div class="relative">
+              <div class="absolute bg-compare-b-smile">
+                <img src="/theme/images/gfx-compare-b-smile.png" alt="smile" width="210px" height="140px">
+              </div>
+            </div>
+
+            <div class="flex flex-wrap justify-center">
+
+              <div class="w-20 mt4">
+                <img src="/theme/images/gfx-compare-b.png" alt="ok" width="75px" height="315px">
+              </div>
+              <div class="w-80 mt4">
+                <p class="ma0 ml3">
+                  Opt in to data collection <i>or decide to not participate at all.</i>
+                </p>
+                <p class="ma0 ml3 mt4">
+                  Review own data <i>with detailed explanations of metrics and terms.</i>
+                </p>
+                <p class="ma0 ml3 mt4">
+                  Only delete usage data <i>or opt out completly at any time.</i>
```

Comment:
Would this be stronger if we flipped it?

"Opt out completely at any time or only delete existing usage data"?

Also, typo in completely.

Reply:
:+1: